include/sharg/validators.hpp

```diff
@@ -632,23 +594,31 @@ class output_file_validator : public file_validator_base
      */
 
     //!\copydoc sharg::input_file_validator::input_file_validator()
-    output_file_validator() : output_file_validator{output_file_open_options::create_new}
-    {}
+    output_file_validator() : output_file_validator{output_file_open_options::create_new} {}
 
-    output_file_validator(output_file_validator const &) = default; //!< Defaulted.
-    output_file_validator(output_file_validator &&) = default; //!< Defaulted.
+    output_file_validator(output_file_validator const &) = default;             //!< Defaulted.
+    output_file_validator(output_file_validator &&) = default;                  //!< Defaulted.
     output_file_validator & operator=(output_file_validator const &) = default; //!< Defaulted.
-    output_file_validator & operator=(output_file_validator &&) = default; //!< Defaulted.
+    output_file_validator & operator=(output_file_validator &&) = default;      //!< Defaulted.
     virtual ~output_file_validator() = default; //!< Virtual Destructor.
 
+    /*!\brief Constructs from a given overwrite mode.
+     * \param[in] mode A sharg::output_file_open_options indicating whether the validator throws if a file already
+                       exists.
+     */
+    explicit output_file_validator(output_file_open_options const mode)
+        : file_validator_base{}, mode{mode}
+    {
+        file_validator_base::extensions = std::move(std::vector<std::string> {});
+        file_validator_base::extensions_str = std::string {};
+    }
+
     /*!\brief Constructs from a given overwrite mode and a list of valid extensions.
      * \param[in] mode A sharg::output_file_open_options indicating whether the validator throws if a file already
                        exists.
-     * \param[in] extensions The valid extensions to validate for. Defaults to
-     *                       sharg::output_file_validator::default_extensions.
+     * \param[in] extensions The valid extensions to validate for.
      */
-    explicit output_file_validator(output_file_open_options const mode,
-                                   std::vector<std::string> extensions = default_extensions())
+    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions)
```

Comment:
I think you can do this: 

```suggestion
    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions = {})
```
Which will construct an empty `std::vector<std::string>{}`. If it doesn't work,
```suggestion
    explicit output_file_validator(output_file_open_options const mode, std::vector<std::string> extensions = std::vector<std::string>{})
```
Will work for sure.

Your solution works well too but this way we only need a single constructor.

Reply:
Sorry that it took so long. The first version already worked :)